action.yaml

```diff
@@ -13,11 +13,12 @@ inputs:
   mix-env:
     required: true
     type: string
+  # booleans are not supported so using a string value
```

Comment:
This makes it seem like booleans aren't supported anywhere. I'd recommend mentioning that they're not supported for workflow dispatches and composite actions and linking to the issue. (if that last comment is accurate)